frank/parameter_descriptions.json

```diff
@@ -12,12 +12,14 @@
 
   "modify_data" : {
     "baseline_range"   : "(Deprojected) baseline range outside of which visibilities are truncated, unit=[\\lambda]",
-    "correct_weights"  : "Whether to estimate the data's weights (as they may be unknown for mock data)"
+    "correct_weights"  : "Whether to estimate the data's weights (as they may be unknown for mock data)",
+    "norm_wle"         : "Wavelength (unit=[m]) by which to normalize the (u, v) points. Not needed if the (u, v) points are already in units of [\\lambda]"
```

Comment:
Can you be explicit that the wavelength is used to convert the uv points to radians? "Normalize" is a bit confusing

Reply:
yeah, fixed in d489249